osu.Game/Beatmaps/WorkingBeatmap.cs

```diff
@@ -17,6 +18,14 @@ public abstract class WorkingBeatmap : IDisposable
 
         public readonly BeatmapSetInfo BeatmapSetInfo;
 
+        /// <summary>
+        /// A play mode that is preferred for this beatmap. This allows for conversion between game modes where feasible,

```

Comment:
The comment should describe what the member actually does, not in such vague terms. What I mean is using something like `PlayMode becomes PreferredPlayMode if conversion is feasible.` instead of `This allows...`